src/main/webapp/sass/_core.scss

```diff
@@ -3076,10 +3076,102 @@ div.border {
 	font-style:italic;
 }
 
-#body .fullWidthAlert {
-	@include border-radius(0px);
-	box-shadow: none;
-	padding: 7px 15px;
+#body .synapseAlert {
+	&.global {
```

Comment:
if it also has the `global` class, then fix it to the bottom-left of the viewport with full width.